### PR TITLE
docs(readme): prettify readme, update downloads stats badge

### LINKED ISSUE #403: Switch downloads badge source to pepy.tech
pypistats.org returns a 502 error in the last week so i think need to migrate to another statistics service like https://pepy.tech/project/aiogram

### PR DESCRIPTION
**Pull request by uwinx**
# Description

readme file editions.

Fixes #403 

## Type of change

- [x] Documentation (typos, code examples or any documentation update)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,71 @@
 
 **aiogram** is a pretty simple and fully asynchronous framework for [Telegram Bot API](https://core.telegram.org/bots/api) written in Python 3.7 with [asyncio](https://docs.python.org/3/library/asyncio.html) and [aiohttp](https://github.com/aio-libs/aiohttp). It helps you to make your bots faster and simpler.
 
-You can [read the docs here](http://docs.aiogram.dev/en/latest/).
+
+<details>
+  <summary>📚 Some simple code examples</summary>
+
+
+**Few steps, before you get started...**
```
Comment (evgfilim1) [inline README.md:21]: Unnecessary comma
```suggestion
**Few steps before you get started...**
```

File: README.md [hunk 2]
```diff
@@ -13,15 +13,71 @@
 
 **aiogram** is a pretty simple and fully asynchronous framework for [Telegram Bot API](https://core.telegram.org/bots/api) written in Python 3.7 with [asyncio](https://docs.python.org/3/library/asyncio.html) and [aiohttp](https://github.com/aio-libs/aiohttp). It helps you to make your bots faster and simpler.
 
-You can [read the docs here](http://docs.aiogram.dev/en/latest/).
+
+<details>
+  <summary>📚 Some simple code examples</summary>
```
Comment (evgfilim1) [inline README.md:18]: I think we should make this block more noticeable

**Review by evgfilim1 — COMMENTED** [review]
Looks nice, but ...

**Comment by uwinx** [comment]
@JrooTJunior, we can close this, since original PyPI stats are back

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 [![\[Telegram\] aiogram live](https://img.shields.io/badge/telegram-aiogram-blue.svg?style=flat-square)](https://t.me/aiogram_live)
 [![PyPi Package Version](https://img.shields.io/pypi/v/aiogram.svg?style=flat-square)](https://pypi.python.org/pypi/aiogram)
 [![PyPi status](https://img.shields.io/pypi/status/aiogram.svg?style=flat-square)](https://pypi.python.org/pypi/aiogram)
-[![Downloads](https://img.shields.io/pypi/dm/aiogram.svg?style=flat-square)](https://pypi.python.org/pypi/aiogram)
+[![Downloads](https://pepy.tech/badge/aiogram)](https://pepy.tech/project/aiogram)
```
Comment (JrooTJunior) [inline README.md:7]:
```suggestion
[![Downloads](https://img.shields.io/pypi/dm/aiogram.svg?style=flat-square)](https://pypi.python.org/pypi/aiogram)
```

**Review by JrooTJunior — APPROVED** [review]

**Pull request merged**